content/rs/databases/active-active/create-active-active.md

```diff
@@ -73,7 +79,7 @@ Every instance of an Active-Active database can receive write operations, and al
         When replication is enabled, every Active-Active database master shard is replicated to a corresponding replica shard. The replica shards are then used to synchronize data between the instances, and the master shards are dedicated to handling client requests.
         We also recommend that you enable [replica HA]({{< relref "/rs/administering/database-operations/replica-ha.md" >}}) to ensure that the replica shards are highly-available for this synchronization.
 
```

Comment:
Similar to "Get started with A-A", maybe change the following headers:
- "Connecting using redis-cli" to "Connect with redis-cli"
- "Connecting using Hello World application in Python" to "Connect with Python"

There are also places where Redis and Python are not capitalized correctly.